app/earn/components/cells/vault-name-cell.tsx

```diff
@@ -20,9 +20,9 @@ export const VaultNameCell: React.FC<Props> = ({ vault }) => {
       <div className={styles.name}>
         {vault.asset.symbol}
         {currentTVL < TVL_THRESHOLD && (
-          <Badge disableOutline size="xs" variant="flat">
+          <CellChip>
             NEW
```

Comment:
Please remove this part for now. I think our vaults pretty long lived. And we sure that this bage not help us. Let's try without it